From 51477567bd408ccb00c6f21af8374b974824f746 Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Fri, 27 Sep 2024 15:16:26 +0200
Subject: [PATCH 14/14] folio: Add FlipFlap

For smartcover
---
 base.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/base.mk b/base.mk
index 59bdf6c..99afa06 100644
--- a/base.mk
+++ b/base.mk
@@ -271,3 +271,7 @@ PRODUCT_PACKAGES += \
 PRODUCT_COPY_FILES += \
     device/phh/treble/folio/init.folio.rc:system/etc/init/init.folio.rc
 
+# FlipFlap Huawei
+PRODUCT_PACKAGES += \
+    FlipFlap
+
-- 
2.25.1

