From aacad002ed85c0dfa77427dba0e314204ba0420e Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Fri, 27 Sep 2024 21:37:44 +0200
Subject: [PATCH 15/15] fmradio: add fmradio for huawei device kirin proc

---
 base.mk | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/base.mk b/base.mk
index 99afa06..7eb1daa 100644
--- a/base.mk
+++ b/base.mk
@@ -275,3 +275,9 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PACKAGES += \
     FlipFlap
 
+# FM
+PRODUCT_PACKAGES += \
+    FMApp2-Hisi \
+    libfm_jni \
+    hisi.fmradio
+
-- 
2.25.1

